data/json/items/melee/swords_and_blades.json

```diff
@@ -246,6 +246,42 @@
     "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 11 ] ],
     "flags": [ "STAB", "SHEATH_KNIFE" ]
   },
+  {
+    "id": "bone_knife",
+    "name": { "str": "bone shiv" },
+    "type": "TOOL",
+    "description": "A femur or other bone, at least 30 cm long, which has been broken at one end and sharpened into a cutting tool. Its jagged edge is wicked but fragile.",
+    "symbol": "/",
+    "color": "white",
+    "weight": 169,
+    "//": "literally 75% of a bone.",
+    "volume": "188 ml",
+    "price": 0,
+    "price_postapoc": 0,
+    "to_hit": -1,
+    "bashing": 1,
+    "cutting": 6,
+    "material": "bone",
+    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", 12 ] ],
+    "flags": [ "STAB", "SHEATH_KNIFE", "FRAGILE_MELEE" ]
+  },
+  {
+    "id": "knife_churl",
+    "type": "TOOL",
+    "name": { "str": "baselard" },
+    "description": "A medieval dagger forged from rough iron. It is not the sharpest tool in the shed, but it is certainly one of the largest.",
+    "weight": "420 g",
+    "volume": "250 ml",
+    "price": 19590,
+    "to_hit": 1,
```

Comment:
base -2
grip: weapon +2
length: short +1
surface: point -2
balance: balanced +1
total is 0

bashing damage should be raised in the area of 3-5, and cutting damage to around 20, to compensate and to match other knives after the melee weapon rebalance.

Reply:
@xanderrootslayer It looks like "knife_baselard" has bashing 5, cutting 20 in your current version. Please confirm and resolve. Thanks!

Reply:
fix'd